### PR TITLE
Fix: Wrong localized string in stringtable.xml

### PR DESCRIPTION
**Pull request by GoldJohnKing**
- ignore changelog

**When merged this pull request will:**
- Fix both Traditional & Simplified Chinese localization string "Delete".
- Fix Traditional & Simplified Chinese and Japanese localization string "Order".
- https://github.com/Vdauphin/HeartsAndMinds/issues/400

**Final test:**
- [x] local
- [x] server

### CODE REVIEW

**Comment by Vdauphin** [comment]
STR_BTC_HAM_ACTION_DATA_DELETE is necessary because https://github.com/GodofMonkeys/Arma-III-Chinese-Localization-Enhanced replace the correct BI Chinese translation from str_a3_cfgvehicles_modulerespawnvehicle_f_arguments_wreck_values_delete_0

Are you agree?

**Comment by 1kuemmel1** [comment]
For 'orders', you could use `STR_BTC_HAM_ACTION_ORDERS_MAIN`

File: =BTC=co@30_Hearts_and_Minds.Altis/core/doc.sqf [hunk 1]
```diff
@@ -41,7 +41,7 @@ player createDiaryRecord [_MainCategory, [localize "str_a3_mdl_category_intel",
 player createDiaryRecord [_MainCategory, [localize "STR_BTC_HAM_DOC_REPUTATION_TITLE", localize "STR_BTC_HAM_DOC_REPUTATION_TEXT"]];
 
 //Orders
-player createDiaryRecord [_MainCategory, [localize "str_3den_waypoint_attribute_order_displayname", localize "STR_BTC_HAM_DOC_ORDERS_TEXT"]];
+player createDiaryRecord [_MainCategory, [localize "STR_BTC_HAM_DOC_ORDERS_TITLE", localize "STR_BTC_HAM_DOC_ORDERS_TEXT"]];
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/core/doc.sqf]:
```suggestion
player createDiaryRecord [_MainCategory, [localize "STR_BTC_HAM_ACTION_ORDERS_MAIN", localize "STR_BTC_HAM_DOC_ORDERS_TEXT"]];
```

And remove the STR_BTC_HAM_DOC_ORDERS_TITLE you added

**Comment by Vdauphin** [comment]
https://github.com/Vdauphin/HeartsAndMinds/pull/758#issuecomment-522335493 ?

**Comment by GoldJohnKing** [comment]
@Vdauphin I could only see a "?" here. What's the problem?

**Comment by Vdauphin** [comment]
STR_BTC_HAM_ACTION_DATA_DELETE is necessary because https://github.com/GodofMonkeys/Arma-III-Chinese-Localization-Enhanced replace the correct BI Chinese translation from str_a3_cfgvehicles_modulerespawnvehicle_f_arguments_wreck_values_delete_0

Are you agree?

https://github.com/GodofMonkeys/Arma-III-Chinese-Localization-Enhanced/blob/85a335f472ef7fa0350856bd039c126a52c913a0/addons/a3/language_f_beta/stringtable.xml#L4210-L4213

:p

**Comment by GoldJohnKing** [comment]
Chinese isn't word to word translation, there're some special gramma we need to obey. So from my point of view, translation in that mod is more accurate in the exact situation, though I cant absolutely say BI's translation is totally wrong.

**Comment by GoldJohnKing** [comment]
I cant say using BI's translation is wrong, and it does suit H&M in this situation, using it might be better for H&M's stringtable to be clean and tidy... It's hard to decide. Isn't there any other "Delete" in game?

**Comment by Vdauphin** [comment]
str_a3_cfgvehicles_modulerespawnvehicle_f_arguments_wreck_values_delete_0 already exist. It is just because https://github.com/GodofMonkeys/Arma-III-Chinese-Localization-Enhanced mod change it to a wrong one from the H&M situation

**Pull request merged**